plugins/floating-pages-exceptions.txt

```diff
@@ -10,4 +10,15 @@
 integrations/language-clients/java/client-v1
 integrations/language-clients/java/jdbc-v1
 integrations/data-ingestion/clickpipes/postgres/maintenance.md
-operations/backup.md
+interfaces/cpp
```

Comment:
The old files still exist but are not assigned to any sidebar now. This tricks the CI into not complaining about this.